extra/modules/fiftyone-devicedetection/src/main/java/org/prebid/server/hooks/modules/fiftyone/devicedetection/core/detection/DeviceRefiner.java

```diff
@@ -0,0 +1,13 @@
+package org.prebid.server.hooks.modules.fiftyone.devicedetection.core.detection;
+
+import org.prebid.server.hooks.modules.fiftyone.devicedetection.core.adapters.DeviceInfoBuilderMethodSet;
+import org.prebid.server.hooks.modules.fiftyone.devicedetection.core.device.DeviceInfo;
+import org.prebid.server.hooks.modules.fiftyone.devicedetection.model.boundary.CollectedEvidence;
+import org.prebid.server.hooks.modules.fiftyone.devicedetection.model.boundary.EnrichmentResult;
+
+public interface DeviceRefiner {
```

Comment:
I believe this interface can be just removed, since I don't see a point having it

Reply:
this was removed